documentation/staging/content/userguide/managing-domains/apply-patched-images.md

```diff
@@ -0,0 +1,58 @@
++++
+title = "Apply patched binary images to a running domain"
+date = 2019-02-23T16:43:45-05:00
+weight = 9
+pre = "<b> </b>"
++++
+
```

Comment:
IIRC, there are detailed discussions about updating images in the 'lifecycle' documentation. If so, perhaps this document belongs as a chapter in that same documentation? Or at least the two sets of doc should cross-reference?

Reply:
I think I'm particularly recalling the I was particular thinking about the CI/CD discussions  https://oracle.github.io/weblogic-kubernetes-operator/userguide/cicd/ , which seem to go into a great amount of detail in this area...

Reply:
I agree that there should be a link to the lifecycle documentation from this page.  The documentation that I believe needs to be merged and refactored with this one is the Base Images https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-in-image/base-images/except for the last section.

Reply:
We should make reference to https://oracle.github.io/weblogic-kubernetes-operator/userguide/cicd/

Reply:
@tbarnes-us @mriccell Let's address these issues, doc content placement and cross-references, in a meeting. Thank you.